src/service/dialogflow/conv.ts

```diff
@@ -183,6 +183,9 @@ export class DialogflowConversation<
   /** @public */
   version: number
 
+  /** @hidden */
+  _followup?: JsonObject
```

Comment:
Can you make this a Union type of [`GoogleCloudDialogflowV2EventInput`](https://github.com/actions-on-google/actions-on-google-nodejs/blob/da4b06e1dc15317dfe405365f4d8f63f9fa13914/src/service/dialogflow/api/v2.ts#L252) `|` [`DialogflowV1FollowupEvent `](https://github.com/actions-on-google/actions-on-google-nodejs/blob/da4b06e1dc15317dfe405365f4d8f63f9fa13914/src/service/dialogflow/api/v1.ts#L210)?

It's more specific and better for type checking. Then you might need to coerce it to the respected type in the serialize code.